src/app/_theme.scss

```diff
@@ -0,0 +1,56 @@
+// Theme variables
+:root {
+  // Base colours
+  --theme-accent-primary: var(--dim-brand); // Interaction styles (hover, active, focus)
+  --theme-accent-secondary: #68a0b7; // Nested interaction (character menu), armor/weapon stat deltas
```

Comment:
As I mentioned before, we should probably split the two meanings here - nested interaction should go away, and armor/weapon stat deltas should either remain un-theme-able, or have a variable specific to that usage. I wouldn't want someone to theme the secondary accent color and have all the mod stat bars change color as a result.

Reply:
Gotcha. Will use a sass var for the stat colour for now… 

In future it's worth considering using a common blue for stat visualisation in the compare view. The theme could be used to define the suite of colours used to highlight various stat differences. 

<img width="573" alt="image" src="https://github.com/DestinyItemManager/DIM/assets/1396158/12d27a93-4906-45b9-9b16-7ef91a794f41">

I've skipped this for now since it appears that the compare sheet colours are injected as inline styles. These would ideally need to be managed as a set of semantic css classes - at the moment they appear to be hard-coded in `formatters.ts`. Atm I must admit I can't figure out what the various colours semantically mean (blue/red seem to mostly be best/worst - but sometimes also green and yellow are for certain stats? then there's also orange..? )

Reply:
I've tried to change/get rid of those colors for a long time but people get mad if you do... they're documented at https://github.com/DestinyItemManager/DIM/wiki/Compare